src/main/java/ninjabrainbot/data/endereye/Throw.java

```diff
@@ -126,8 +155,14 @@ public double getStd() {
 	}
 
 	@Override
-	public void addCorrection(double angle) {
-		correction += angle;
+	public void addCorrection(int multiplier, NinjabrainBotPreferences preferences) {
+		double change = 0.01;
+		if (preferences.useTallRes.get()) {
+			final double toRad = Math.PI / 180.0;
+			change = Math.atan(2 * Math.tan(15 * toRad) / preferences.resolutionHeight.get()) / Math.cos(beta * toRad) / toRad;
+		}
```

Comment:
Yes, users will correct once for each pixel off.
beta is used in case users tower up after throwing (but before measuring) or throw during a jump, I'm not too concerned about the small changes that occur when the eye bobs up and down.